build-logic/gradle-functional-testing/src/main/groovy/org.graalvm.build.functional-testing.gradle

```diff
@@ -75,14 +75,14 @@ gradlePlugin.testSourceSets(sourceSets.functionalTest)
 configurations.functionalTestImplementation.extendsFrom(configurations.testImplementation)
 
 def graalVm = javaToolchains.launcherFor {
-    languageVersion.set(JavaLanguageVersion.of(11))
-    vendor.set(JvmVendorSpec.matching("GraalVM"))
+    languageVersion.set(JavaLanguageVersion.of(17))
+//    vendor.set(JvmVendorSpec.matching("Oracle Corporation"))
```

Comment:
Intentionally left so that we figure out what to do with this : the build will fail if it's not an Oracle GraalVM, but there's no way to make it work properly if we set `matching("Oracle Corporation")` since it could select a different implementation